public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx

```diff
@@ -170,7 +170,9 @@ export const HeatmapHoverView = ({ data, hover, showHistogram }: Props) => {
         <div>
           Bucket: {yDisp(yBucketMin)} - {yDisp(yBucketMax)}
         </div>
-        <div>Count: {count}</div>
+        <div>
+          {getFieldDisplayName(countField!, data.heatmap)}: {count}
```

Comment:
can we memo this?

Reply:
Maybe... But also not expensive -- the name is pulled from the cache on fiels

Reply:
What is the meaning of `1e-06` here? 

<img width="556" alt="image" src="https://user-images.githubusercontent.com/89186/197683559-982cd8d0-f5b4-4132-98d4-7627db04e926.png">